### DISCUSSION TITLE
Coupled runs 37 and 40. New enthalpy formulation

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#383] [Discussion about coupled runs]
Set of simulations to assess the new enthalpy formulation.

- Coupled run "37" (old enthalpy): #367
- Coupled run "40" (new enthalpy): #379

___

Slide for CESM co-chair meeting (08/29/2023)
![Screen Shot 2023-08-29 at 2 44 26 PM](https://github.com/NCAR/amwg_dev/assets/9723220/b26146dd-31b9-48b6-8ebd-da1a6a7cee54)

___

![Screen Shot 2023-08-29 at 2 46 25 PM](https://github.com/NCAR/amwg_dev/assets/9723220/36c40660-ae41-4a45-bddb-4befbe297621)

@PeterHjortLauritzen, @JulioTBacmeister, @adamrher, @gustavo-marques, @tto061

**Comment by adamrher** [comment]
W.Pac SSTs in the runs:

![temp_tseries_SST](https://github.com/NCAR/amwg_dev/assets/29961476/5d03baab-3f2a-4d54-9d50-4224446d3f1e)

This is just a replication of Thomas' plots. The hypothesis is that the W.Pac cooling is instigating labsea freeze through teleconnections. The 37 and 40 runs are noticeably a lot warmer in W.Pac, and no sign of labsea freezing yet. Here's the same plot but extended to year 215 so that all over 26g is shown.

![temp_tseries_SST](https://github.com/NCAR/amwg_dev/assets/29961476/ef6d41c5-1765-4e81-88f8-d850508a728b)

There is local minimum of W.Pac cooling when 26g starts to freeze over, similar to run 35 (although there are cooler local min's earlier in the sim that don't freeze labsea; it's possible that I'm looking at to narrow a region of W.Pac for informing teleconnections).

@cecilehannay two questions:

1.  Can we extend 38 to see if it freezes over?
2. I had thought we were going to experiment with enthalpy fluxes using run 35. The idea is that we thought the enthalpy fluxes could avoid a labsea freeze, which 35 does at year ~30. 37 still hasn't frozen over. Why are we using 37 to test the enthalpy fluxes?

I'll be adding two more run-requests to test a parameter in the ice cloud fraction. I'll be requesting 38 as a baseline. I'd like to use 38 as a baseline for future enthalpy flux experiments too, but we can talk about this in-person tmrw.

**Reply by cecilehannay** [reply]
1. We can continue with run **38**. Please note that this run has a RESTOM=-0.8 W/m2 

2. One of the reason we wanted to continue **37** was that we wanted to have a configuration ready - in case solving the current questions with the new enthalpy takes an extended amount of time. Essentially, 37 represented the previous approach of the enthalpy and in all the runs with the old enthalpy, it had the more realistic RESTOM value of 0.3W/m2.  

3. We selected **37** over **35** for testing the new enthalpy because we were concerned that an enthalpy run based on **35** could result in an excessively negative RESTOM. Run 35 had a RESTOM=-0.8W/m2 and the new enthalpy decreases further the RESTOM by -0.6W/m2. Therefore, we anticipated that a run based on 35 + new enthalpy would yield to a RESTOM=-1.4W/m2

**Comment by cecilehannay** [comment]
Here is an updated timeseries for 37 and 40:
![037_040_26g_updated](https://github.com/NCAR/amwg_dev/assets/9723220/230e2916-7f0c-49e6-8542-033a136840ee)

**Comment by cecilehannay** [comment]
@PeterHjortLauritzen:
We have now two one-year runs with the `thermo_budget_history`.  
- `037_thermo` (#385)
- `040_thermo` (#386)
These runs were starting from `037` and `040` at year `15`.

The output is in the files `h7` on derecho:
```
/glade/derecho/scratch/hannay/archive/b.e23_alpha16b.BLT1850.ne30_t232.037_thermo/atm/hist
/glade/derecho/scratch/hannay/archive/b.e23_alpha16b.BLT1850.ne30_t232.040_thermo/atm/hist
```
